src/tools/wast2wasm.cc

```diff
@@ -238,5 +238,5 @@ int main(int argc, char** argv) {
 
   wabt_destroy_ast_lexer(lexer);
   wabt_destroy_script(&script);
-  return result;
+  return static_cast<int>(result);
```

Comment:
Return result != OK ?  Or are there different result values you want to return from main?